irods/manager/data_object_manager.py

```diff
@@ -151,6 +151,47 @@ def move(self, src_path, dest_path):
             conn.send(message)
             response = conn.recv()
 
+    def copy(self, src_path, dest_path):
+        # check if dest is a collection
+        # if so append filename to it
+        if self.sess.collections.exists(dest_path):
+            filename = src_path.rsplit('/', 1)[1]
+            target_path = dest_path + '/' + filename
+        else:
+            target_path = dest_path
+
+        src = FileOpenRequest(
+            objPath=src_path,
+            createMode=0,
+            openFlags=0,
+            offset=0,
+            dataSize=0,
+            numThreads=0,
+            oprType=11,   # RENAME_DATA_OBJ
```

Comment:
I'm unsure if there is a more appropriate `oprType` to use for copy (instead of rename). Couldn't find a reference anywhere.